Jenkinsfile

```diff
@@ -0,0 +1,58 @@
+
+def testReport = 'test-report.xml'
+def stagingBucket = 'internal.clyqz.com'
+def stagingPrefix = '/docs/whotracksme'
+def productionBucket = 'cliqz-tracking-monitor'
+def productionPrefix = ''
+
+node('docker') {
+    stage ('Checkout') {
+        retry(5) {
+            checkout scm
+        }
+    }
+    def img
+
+    stage('Build Docker Image') {
+        img = docker.build('whotracksme', '.')
+    }
+
+    img.inside('-u 0:0') {
+        try {
+
+            stage('Install') {
+                sh("pip install '.[test,website]'")
+            }
+
+            stage('Test') {
+                try {
+                    sh(script: "pytest --junit-xml=${testReport}")
+                } catch(err) {
+                    junit(testReport)
+                    currentBuild.result = "FAILURE"
+                }
+            }
+
+            stage('Build site') {
+                sh('whotracksme website')
+            }
+
+            stage('Publish Site') {
+                def deployArgs = ''
+                if (env.BRANCH_NAME.contains('PR')) {
+                    deployArgs = "${stagingBucket} ${stagingPrefix}/${env.BRANCH_NAME}"
+                } else if (env.BRANCH_NAME == 'production') {
+                    deployArgs = "${productionBucket} ${productionPrefix} --production"
+                } else {
+                    deployArgs = "${stagingBucket} ${stagingPrefix}/latest"
+                }
+                sh("python deploy_to_s3.py ${deployArgs}")
```

Comment:
Looks good, but as you suggested at some point, maybe we could make this part of the `whotracksme` entrypoint?

```sh
$ whotracksme website deploy --bucket BUCKET [--prefix PREFIX] [--production]
```

Reply:
Yes, could be. It depends if third-party users are also going to use this script. At the moment it is fairly specific deployment, for our infrastructure. The `whotracksme` command is more to enable people to deploy their own local copy, rather than their own hosted version of the site.